resources/quality/ultimaker_s5/um_s5_aa0.4_CPE_Draft_Print.inst.cfg

```diff
@@ -17,13 +17,12 @@ material_initial_print_temperature = =material_print_temperature - 5
 material_final_print_temperature = =material_print_temperature - 10
 retraction_combing_max_distance = 50
 retraction_prime_speed = =retraction_speed
-skin_edge_support_thickness = =0.8 if infill_sparse_density < 30 else 0
 skin_overlap = 20
 speed_print = 60
 speed_layer_0 = =math.ceil(speed_print * 20 / 60)
 speed_topbottom = =math.ceil(speed_print * 35 / 60)
 speed_wall = =math.ceil(speed_print * 45 / 60)
 speed_wall_0 = =math.ceil(speed_wall * 35 / 45)
-top_thickness = =0.8 if infill_sparse_density < 30 and skin_edge_support_thickness == 0.8 else top_bottom_thickness
+top_thickness = =0.8 if infill_sparse_density < 30 else top_bottom_thickness
```

Comment:
Is this needed? Draft mode would normally have a top thickness of 1.2mm iso 0.8mm. This is thus an productivity improvement that is created to compensate for the extra time needed to print the skin edge support. 
However I think we can reduce the top (and bottom thickness) in general (and for lower densities the skin edge support prevents it from collapsing), but this should be a separate ticket.
I would remove these top_thickness entries for CPE & PETG for now. It increases print time, but improves the clarity. [PP-23 could be used to optimize this in general](https://ultimaker.atlassian.net/browse/PP-23)

Reply:
I wanted to do the least changes that are really related to PP-15, but I agree this is a weird statement anyways. I will remove it for now.